src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -79,28 +75,23 @@ extern "C" {
 // Trampolined callback types
 CHIP_ERROR pychip_DeviceController_DriveIO(uint32_t sleepTimeMS);
 
-CHIP_ERROR pychip_DeviceController_NewDeviceController(chip::DeviceController::ChipDeviceController ** outDevCtrl);
-CHIP_ERROR pychip_DeviceController_DeleteDeviceController(chip::DeviceController::ChipDeviceController * devCtrl);
+CHIP_ERROR pychip_DeviceController_NewDeviceController(chip::Controller::DeviceCommissioner ** outDevCtrl);
+CHIP_ERROR pychip_DeviceController_DeleteDeviceController(chip::Controller::DeviceCommissioner * devCtrl);
```

Comment:
Here and throughout, it's a little weird to have the variable of type "DeviceCommissioner" be called "devCtrl".  Worth finding better names?

Reply:
@erjiaqing ?